### DISCUSSION TITLE
Infill Patterns

### DISCUSSION THREAD

**Discussion by FozzyMaker** [supermerill/SuperSlicer#2036] [Q&A]
I titled this discussion so other questions about infill can be placed in the thread if desired.

**What is the difference between Monotonic (filled) and Monotonic infill?** I am having a hard time finding documentation on the difference between the two. If there is documentation about this stuff, will you redirect me to there? Thanks!
![image](https://user-images.githubusercontent.com/86342343/145607994-aea6ea9a-b29d-46db-82c7-4d8803fb34ad.png)

**Comment by supermerill** [comment]
A (filled) pattern is a pattern that will add some gap-fill (thin extrusions) where there is still some voids. It will also directly fill surfaces thinner than ~the extrusion width directly with gap-fill.

When you don't want holes in your solid surface, use it.
When you're concerned about printing time and the number of travels, choose the normal one.
